gdk/gdk.device.lisp

```diff
@@ -1491,6 +1558,26 @@ get_device_settings (GdkDevice *device)
 ;;; Since 3.0
 ;;; ----------------------------------------------------------------------------
 
+(defcfun ("gdk_device_get_axis_value" %gdk-device-get-axis-value) :boolean
+  (device (g-object gdk-device))
+  (axes (:pointer :double))
+  (axis-label gdk-atom-as-string)
+  (value (:pointer :double)))
+
+(defun gdk-device-get-axis-value (device axes axis-label)
+  (assert (= (gdk-device-n-axes device) (length axes)))
```

Comment:
Will it otherwise fail hard? I'd leave it out if it was just convenience.

In any case, I have the feeling that the different implementations treat `assert` very differently, at least on CCL the debugger frontend for it is pretty awful and doesn't tell you that much. Not that I have a better solution right now except raising an error manually.

Reply:
This check needs to be there, because the behavior can be undetermined without it.

I used an `assert` because it's what `gdk-device-get-axis` already does. But I'm ok to convert it to an error (and BTW to do the same in `gdk-device-get-axis`).

Reply:
Okay if we already have the `assert` elsewhere leave it I guess. I'll add an issue about error reporting though.